sovrin_client/test/cli/test_pool_upgrade_validation.py

```diff
@@ -0,0 +1,24 @@
+import pytest
+
+from sovrin_node.test.upgrade.conftest import validUpgrade
+from sovrin_client.test.cli.constants import INVALID_SYNTAX
+from plenum.common.util import randomString
+from sovrin_common.constants import JUSTIFICATION
+
+
+def testPoolUpgradeHasInvalidSyntaxIfJustificationIsEmpty(be, do, validUpgrade, trusteeCli):
+    validUpgrade[JUSTIFICATION] = ''
```

Comment:
Is it intentional to keep `JUSTIFICATION` mandatory? If yes then `JUSTIFICATION` should be neither `optional` not `nullable` in `ClientPoolUpgradeOperation`.